server/publications/collections/orders.js

```diff
@@ -1,13 +1,63 @@
 import { Meteor } from "meteor/meteor";
 import { check, Match } from "meteor/check";
 import { Roles } from "meteor/alanning:roles";
-import { Counts } from "meteor/tmeasday:publish-counts";
+import { ReactiveAggregate } from "./reactiveAggregate";
 import { Orders } from "/lib/collections";
 import { Reaction } from "/server/api";
 
+
 /**
- * orders
+ * A shared way of creating a projection
+ * @param {String} shopId - shopId to filter records by
+ * @param {Object} sort - An object containing a sort
+ * @param {Number} limit - An optional limit of how many records to return
+ * @returns {Array} An array of projection operators
  */
+export function createAggregate(shopId, sort = { createdAt: -1 }, limit = 0) {
```

Comment:
is this just for creating an orders aggregation?
Would createOrdersAggregate be more descriptive or is it generic enough to be used for more publications?
In that case would it be better for this to live in another file?